tests/TestCase/Listener/ApiQueryLogListenerTest.php

```diff
@@ -198,6 +198,71 @@ public function testSetupLogging()
         $Instance->setupLogging(new Event('something'));
     }
 
+    /**
+     * Test setting up only specific query loggers
+     *
+     * @return void
+     */
+    public function testSetupLoggingConfiguredSources()
+    {
+        $methodName = 'enableQueryLogging';
+        if (version_compare(Configure::version(), '3.7.0RC', '<')) {
+            $methodName = 'logQueries';
+        }
+
+        $DefaultSource = $this
+            ->getMockBuilder('\Cake\Database\Connection')
```

Comment:
Please `Connection::class` instead of string, same for other classes.

Reply:
Haha I just copied it from an existing test case above. Will update both.

Reply:
I you want you can update existing ones too :slightly_smiling_face:

Reply:
Yeah that is the plan. Going to look through the plugin and fix them.